flaskr/__init__.py

```diff
@@ -32,7 +32,7 @@ def create_app(test_config=None):
     except OSError:
         pass
 
-    @babel.localeselector
+    @babel.localeselector()
```

Comment:
Hm, this still causes an issue.
File "/Users/msmith/Documents/GitHub/flask/.venv/lib/python3.12/site-packages/flask/cli.py", line 72, in find_best_app
    app = app_factory()
          ^^^^^^^^^^^^^
  File "/Users/msmith/Documents/GitHub/flask/flaskr/__init__.py", line 35, in create_app
    @babel.localeselector()
     ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Babel' object has no attribute 'localeselector'

Might this have a possible solution?
https://stackoverflow.com/questions/75270265/attributeerror-babel-object-has-no-attribute-localeselector